edx_rest_framework_extensions/auth/jwt/tests/test_decoder.py

```diff
@@ -68,22 +68,24 @@ def test_valid_token_multiple_valid_issuers(self, jwt_issuer):
 
     def test_failure_invalid_issuer(self):
         """
-        Verifies the function logs decode failures, and raises an InvalidTokenError if the token cannot be decoded
+        Verifies the function logs decode failures with invalid issuer,
+        and raises an InvalidTokenError if the token cannot be decoded
         """
 
         # Create tokens using each invalid issuer and attempt to decode them against
         # the valid issuers list, which won't work
         with mock.patch('edx_rest_framework_extensions.auth.jwt.decoder.logger') as patched_log:
             with self.assertRaises(jwt.InvalidTokenError):
                 self.payload['iss'] = 'invalid-issuer'
-                signing_key = 'invalid-secret-key'
+
                 # Generate a token using the invalid issuer data
-                token = generate_jwt_token(self.payload, signing_key)
+                token = generate_jwt_token(self.payload, None)
```

Comment:
It will pick the `settings.JWT_AUTH['JWT_ISSUERS'][0]['SECRET_KEY']` in case of None.

Reply:
What do you think of the following, which somewhat incorporates your comment?
```
# signing key of None will use the default valid signing key
valid_signing_key = None
# Generate a token using the invalid issuer data
token = generate_jwt_token(self.payload, valid_signing_key)
```